edge/pkg/edged/edged_getters.go

```diff
@@ -26,6 +26,8 @@ package edged
 import (
 	"fmt"
 	"io/ioutil"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/kubelet/config"
```

Comment:
please segregates different category of imports through blank line, and check all the files

Reply:
Have used goimport...probably since there is only one import in each category may be its not put a line